test/utils.dart

```diff
@@ -132,7 +132,8 @@ class _HrxTestFiles {
             "given test.";
       }
     } else if (filename == "arguments") {
-      arguments = contents.trim().split(" ");
+      // TODO(jathak): Handle spaces with quotes here.
```

Comment:
Not for the first time, I really wish Dart had a shellparse package. In this case, I think you should be able to make do with scanning for instances of `"[^"]+"|[^"\s][^\s]*`

Reply:
Done.